server/index.js

```diff
@@ -13,16 +13,16 @@ app.all('*', function (req, res, next) {
 
 app.get('*', async (req, res) => {
   /** @type {String} */
-  const code = req.query.code;
+  const {code, client_id, client_secret, serverURL} = req.query;
   try {
     console.log(`Authenticating Code: ${code.substr(0, 3)}...`);
     const response = await axios.post(
-      'https://github.com/login/oauth/access_token',
+      serverURL || 'https://github.com/login/oauth/access_token',
       {},
       {
         params: {
-          client_id: process.env.OAUTH_CLIENT_ID,
-          client_secret: process.env.OAUTH_CLIENT_SECRET,
+          client_id: client_id || process.env.OAUTH_CLIENT_ID,
```

Comment:
If I only send a `serverURL`, Issueist will expose `process.env.OAUTH_CLIENT_ID` and `process.env.OAUTH_CLIENT_SECRET` to me

Reply:
This was a brilliant catch